src/liveEditor/components/emptyBlock.tsx

```diff
@@ -0,0 +1,43 @@
+import { CslpData } from "../../cslp/types/cslp.types";
+import { VisualEditorCslpEventDetails } from "../types/liveEditor.types";
+import liveEditorPostMessage from "../utils/liveEditorPostMessage";
+import { LiveEditorPostMessageEvents } from "../utils/types/postMessage.types";
+
+interface EmptyBlockProps {
+  details: {
+    editableElement: Element;
+    cslpData: string;
+    fieldMetadata: CslpData;
+  }
+}
+
+export function EmptyBlock(props: EmptyBlockProps) {
+
+  const { details } = props
+  
+  const blockParentName = details.fieldMetadata.fieldPath.split('.').at(-1)!.split('_').join(' ')
+  const capitalizedBlockParentName = blockParentName.replace(/\b[a-z]/g, (letter: string) => {
+    return letter.toUpperCase();
+  });
```

Comment:
There should be a better way to retrieve this value than deriving and capitalizing it. It looks like it can break in the future.

Reply:
Yes, we can get it from schema. I just implemented it in the last commit, please check.